pkgs/by-name/cp/cpu-energy-meter/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libcap
+}:
+stdenv.mkDerivation rec {
+  pname = "cpu-energy-meter";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "sosy-lab";
+    repo = "cpu-energy-meter";
+    rev = version;
+    hash = "sha256-QW65Z8mRYLHcyLeOtNAHjwPNWAUP214wqIYclK+whFw=";
+  };
+
+  patchPhase = ''
+    substituteInPlace Makefile \
+      --replace "DESTDIR :=" "DESTDIR := $out" \
+      --replace "PREFIX := /usr/local" "PREFIX :="
+  '';
+
+  buildInputs = [ libcap ];
+
+  env.NIX_CFLAGS_COMPILE = "-fcommon";
+
+  postInstall = ''
+    install -D --mode=444 --target-directory=$out/etc/udev/rules.d \
+      $src/debian/additional_files/59-msr.rules
```

This could be shortened down if you don't want it taking 2 lines.

```suggestion
    install -Dm444 -t $out/etc/udev/rules.d $src/debian/additional_files/59-msr.rules
```